workflows/command_line_tools/SSU-from-tablehits.cwl

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: Workflow
+
+inputs:
+ table_hits: File
+
+outputs:
+ SSU_coordinates: { type: File, outputSource: extract_coords/SSUs_with_coords }
+
+steps:
+  grep:
+    run:
+      class: CommandLineTool
+      inputs:
+        hits: { type: File, streamable: true }
+      baseCommand: [ grep, SSU ]
+      arguments: [ $(inputs.hits.path) ]
+      outputs: { SSUs: { type: stdout } }
+    in: { hits: table_hits }
+    out: [ SSUs ]
+
+  extract_coords:
+    run:
+      class: CommandLineTool
+      inputs:
+        SSUs: { type: File, streamable: true }
+      baseCommand: awk
+      arguments:
+        - '{print $4"-"$3"/"$10"-"$11" "$10" "$11" "$4}'
```

Comment:
From reading http://eddylab.org/infernal/Userguide.pdf#page=60
and `esl-sfetch -h`
```
[...]
  In retrieving subsequences listed in a file (-C -f, or just -Cf), each line of the file
  is in GDF format: <newname> <from> <to> <source seqname>, space/tab delimited.
```
I'm guessing the format is as follows:

`query name`—`accession`**/**`seq from (ali coord)`—`seq to (ali coord)` `seq from (ali coord)` `seq to (ali coord)` `query name`

where `query name`—`accession`**/**`seq from (ali coord)`—`seq to (ali coord)` (a.k.a `$4"-"$3"/"$10"-"$11"`) becomes the newname and `seq from (ali coord)` (a.k.a. `$10`), `seq to (ali coord)` (a.k.a `$11`), and `query name` (a.k.a `$4`) are the starting coordinates, ending coordinates, and the original name (respectively).

Shall I document the above in the CWL description?